src/index.js

```diff
@@ -223,6 +224,35 @@ export default ({ apiKey = null, client = null } = {}) => {
                 })
                 .catch(callback);
         },
+        
+        /**
+         * create write stream
+         *
+         * @param {String} token
+         * @param {String} remotePath
+         * @returns {Stream}
+         */
+        createWriteStream(filepath) {
+            return dropboxStream.createDropboxUploadStream({
+                token: client.accessToken,
+                filepath,
+                chunkSize: 1000 * 1024
```

Comment:
Maybe allow setting the `chunkSize` through an `options` object?

Reply:
[fs.createWriteStream](https://nodejs.org/dist/latest-v9.x/docs/api/fs.html#fs_fs_createwritestream_path_options) doesn’t supports `chunkSize`

Reply:
Doesn't forbid us to allow it as an option too...